src/bliss/util/config.h

```diff
@@ -0,0 +1,38 @@
+#ifndef CONFIG_H
+#define CONFIG_H
```

Comment:
Similar to the comment before, `BLISS_CONFIG_H`

Reply:
fixed